hugo/content/kommunen/hamburg.md

```diff
@@ -28,21 +30,79 @@ Links im Bild siehst du die bisherigen CO₂-Emissionen Hamburgs. Was schätzt d
 
-Vielleicht verstehst du jetzt besser, warum so viele Menschen auf die Straße gehen und für Klimaschutz demonstrieren.
+### Was unternimmt Hamburg?
+
+Die Stadt Hamburg beschreibt vier Transformationspfade um die CO₂-Emission zu senken.
+
+**Wärmewende inkl. Gebäudeeffizienz**
+
+Grundsätzlich soll die Effizienz der Beheizungsstruktur und Warmwasserbereitung gesteigert werden.
+
+Außerdem werden unter anderem folgende Maßnahmen ergriffen:
+
+- Steigerung der Anteile erneuerbarer Energien in der Wärmeversorgung
+- Schaffung von Anreizen für den Einsatz hybrider Wärmeversorgungssysteme
+- Neue Wärmenetze mit Einsatz erneuerbarer Energien und Abwärme für Neubau- und Bestandsgebiete
+- Ausbau der Energieberatung
+- Entwicklung einer Hamburger Holzbau-Strategie
+- Umsetzung des Leitfadens Nachhaltiges Bauen des Bundes für öffentliche Gebäude
+- und viele mehr
+
+**Mobilitätswende**
+
+Schwerpunkt ist die Strategie zum Ausbau des ÖPNV.
+
+Außerdem werden unter anderem folgende Maßnahmen ergriffen:
+
+- Förderung des Radverkehrs
+- Verbesserung des Verkehrsmanagements
+- Elektrifizierung von Fahrzeugflotten
+- Ausbau der Ladeinfrastruktur für Elektrofahrzeuge 
+- Ausbau der Landstrominfrastruktur für alle Bereiche der Seeschifffahrt 
+- und viele mehr
+
+**Wirtschaft**
+
+Zu den Schwerpunkten zählen die Beratung und Weiterbildung von Unternehmen, die Weiterentwicklung von Förderprogrammen, sowie eine flexible Anpassung der Energieverbräuche und der Energieeigenerzeugung.
+
+Außerdem werden unter anderem folgende Maßnahmen ergriffen:
+
+- Aufbau eines Netzwerkes Wasserstoffwirtschaft
+- Verstetigung und Förderung des Energieforschungsverbundes Hamburg
+- Ausbau der Kooperation mit Industrie und Handwerk
+- Sichtbarmachung von CO₂-Reduzierungen von Unternehmen ("Clean:tech inside")
+- Einstellung von Klimaschutzmanagerinnen und -managern in jedem Bezirk
+- Flächendeckende Bewerbung von Umwelt-, Klimaschutz- und Klimaanpassungsmaßnahmen in kleinen und mittleren Unternehmen
+- Einwerben von EU-Fördermitteln für Hamburger Klimaschutzprojekte
+- Nutzbarmachung industrieller Abwärme für Wärmenetze
+- und viele mehr
+
+**Klimaanpassung**
+
+Hier geht es weniger um das direkte Einsparen von CO₂. Das übergeordnete Ziel besteht darin, Hamburg zu einer klimaresilienten Stadt zu entwickeln. Oder anders ausgedrückt, Hamburg soll widerstandsfähiger gegenüber den Folgen des Klimawandels werden.
+
+Unter anderem werden folgende Maßnahmen ergriffen:
+
+- Berücksichtigung des Meeresspiegelanstieges beim Küstenhochwasserschutz
+- Berücksichtigung der Auswirkungen der Klimaänderungen beim Binnenhochwasserschutz
+- Sicherung der Trinkwasserversorgung bei verändertem Niederschlagsgeschehen 
+- Sicherung der Abwasserbeseitigung bei Extremwetterereignissen
+- Enntwicklung neuer Strategien für Katastrophenvorsorge und Katastrophenschutz
+- und viele mehr
+
 
 ### Was kann ich tun?
 
```

Comment:
in der nächsten Aufzählung sind Leerzeilen zwischen den Punkten, siehe https://deploy-preview-65--klimawatch.netlify.com/kommunen/hamburg/
Kann man machen, führt aber zu mehr Abstand. Musst Du nicht ändern, ich wollte aber drauf hingewiesen haben.

Reply:
Das ist extra so gewollt. Fand ich hübscher :D

Reply:
Dann ist ja alles gut :)